plugins/backstage-plugin-flux/dev/index.tsx

```diff
@@ -174,6 +183,89 @@ class StubKubernetesClient implements KubernetesApi {
       } as Response;
     }
 
+    if (!init?.method && path === NAMESPACES_PATH) {
+      if (clusterName === 'mock-cluster-1') {
+        return {
+          ok: true,
+          json: () =>
+            Promise.resolve({
+              kind: 'NamespacesList',
+              apiVersion: 'meta.k8s.io/v1',
+              items: [
+                {
+                  name: 'flux-system',
+                  uid: '1dcca7cb-c651-4a86-93b4-ecf440df2353',
+                  resourceVersion: '1583',
+                  creationTimestamp: '2023-10-19T16:34:12Z',
+                  labels: {
+                    'app.kubernetes.io/instance': 'flux-system',
+                    'app.kubernetes.io/part-of': 'flux',
+                    'app.kubernetes.io/version': 'v2.0.0',
+                    'kubernetes.io/metadata.name': 'flux-system',
+                    'kustomize.toolkit.fluxcd.io/name': 'flux-system',
+                    'kustomize.toolkit.fluxcd.io/namespace': 'flux-system',
+                    'pod-security.kubernetes.io/warn': 'restricted',
+                    'pod-security.kubernetes.io/warn-version': 'latest',
+                  },
+                },
+              ],
+            }),
+        } as Response;
+      }
+      if (clusterName === 'mock-cluster-2') {
+        return {
+          ok: true,
+          json: () =>
+            Promise.resolve({
+              kind: 'NamespacesList',
+              apiVersion: 'meta.k8s.io/v1',
+              items: [
+                {
+                  name: 'default',
```

Comment:
Namespace here should also follow k8s ns pattern!

with `{ metadata: { name, labels } }`